ui/app/components/ui/truncated-definition-list/truncated-definition-list.js

```diff
@@ -0,0 +1,74 @@
+import { pick } from 'lodash'
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { COLORS, SIZES } from '../../../helpers/constants/design-system'
+import Box from '../box'
+import Button from '../button'
+import DefinitionList from '../definition-list/definition-list'
+import Popover from '../popover'
+
+export default function TruncatedDefinitionList({
+  dictionary,
+  tooltips,
+  prefaceKeys,
+  title,
+}) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false)
+
+  return (
+    <div className="truncated-definition-list">
```

Comment:
Nit: Could this `div` be replaced by a fragment? It seems like it's just used to group the box and the popover, and we don't need a div for that. The popover ends up in a portal anyway.

Reply:
Yes, in fact [it was a fragment](https://github.com/MetaMask/metamask-extension/pull/10292/commits/c8f1c8fb967b098c672cb488a8375f4baa79c4cd#diff-187fe6a9307da5dc2fbd82de0b7eb8dd1d48354c372319669e3b822a63ce5cecR19) and only got upgraded to the div to hold the className, but that doesn't even matter and I should have stuck with guns.

Reply:
resolved in latest